custom-words.txt

```diff
@@ -3931,4 +3931,6 @@ capacityreservationgroup
 capacityreservationgroups
 ciam
 CIAM
+protobuf
+proto
```

Comment:
do you actually need 'proto' or only 'protobuf'?

Reply:
We have both AFAIK

Reply:
@tomkerkhove Why? I don't think its a real word. Its just a prefix. Please don't add it in custom-words. Consdier adding it in cSpell intead!

Reply:
It's a commonly used term in gRPC space